primevue-gen/build.js

```diff
@@ -0,0 +1,29 @@
+const esbuild = require("esbuild")
+
+const myPlugin = {
+  name: "my-plugin",
+  setup(build) {
+    build.onResolve({ filter: /^vue$/ }, (args) => ({
+      path: args.path,
+      namespace: "my-plugin",
+    }))
+
+    build.onLoad({ filter: /.*/, namespace: "my-plugin" }, () => {
+      const contents = "module.exports = Vue"
+      return { contents }
+    })
+  },
+}
+
+;(async () => {
+  await esbuild.build({
+    entryPoints: ["./src/main.js"],
+    bundle: true,
+    outdir: "../promgen/static/primevue",
+    logLevel: "info",
+    loader: {
+      ".woff2": "dataurl",
+    },
+    plugins: [myPlugin],
+  })
+})()
```

Comment:
A few general comments about this file (`primevue-gen/build.js`):
- Plenty of semi-colons missing; Please add them.
- Some semi-colons are misplaced (look at the begging of the `async` line).
- The file is missing a new-line character. Please add it.
- Has this file been written following some kind of documentation about how to write your build script for esbuild? If so, it would be nice to include a link at the top of this file in case we need to modify it in the future.

Reply:
> - Plenty of semi-colons missing; Please add them.
> - Some semi-colons are misplaced (look at the begging of the async line).
> - The file is missing a new-line character. Please add it.

I see you have addressed this comment with 43381cccbaa8ee644d3c72c043c22c552fec6586.

> - Has this file been written following some kind of documentation about how to write your build script for esbuild? If so, it would be nice to include a link at the top of this file in case we need to modify it in the future.

I see you have addressed this comment with 3a760cdf029d75a44d089bf67dda4f2a4314f48e.

This conversation is now resolved.